### PR TITLE
test(Picker): enhance some tests

### PR DESCRIPTION
**Pull request by xiaoyao96**

### CODE REVIEW

File: src/components/picker/tests/picker.test.tsx [hunk 1]
```diff
@@ -1,9 +1,28 @@
 import React, { createRef, useState } from 'react'
-import { render, fireEvent, waitFor } from 'testing'
+import { render, fireEvent, waitFor, screen } from 'testing'
 import { basicColumns } from '../demos/columns-data'
 import Picker, { PickerRef } from '..'
 import Button from '../../button'
 
+const classPrefix = 'adm-picker'
+
+function waitElementVisibleChange(
```
Comment (yangdepp) [inline src/components/picker/tests/picker.test.tsx:9]: 这个可以抽一个方法出来放在utils，很多基于Picker实现的组件测试都需要用

File: src/components/picker/tests/picker.test.tsx [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
+  fn: () => HTMLElement | null,
+  visible: boolean
+) {
+  expect(fn()).toBeInTheDocument()
+  return waitFor(async () => {
```
Comment (awmleer) [inline src/components/picker/tests/picker.test.tsx:14]: 这个 `async` 和 `waitFor` 好像没有作用……？

Comment (xiaoyao96) [reply src/components/picker/tests/picker.test.tsx:14]: https://testing-library.com/docs/dom-testing-library/api-async/#waitfor

> If you return a promise in the waitFor callback (either explicitly or implicitly with async syntax), then the waitFor utility will not call your callback again until that promise rejects. This allows you to waitFor things that must be checked asynchronously.

waitFor的回调若reject，会循环重试，直到waitFor超时。这里利用了这一点，来等待弹框显示或隐藏。

**Comment by xiaoyao96** [comment]
> 感觉写复杂了，jest-dom 有提供 style 的 matcher 的
> 
> ```js
> const el = document.querySelectorAll(`.adm-picker-popup`)[0]
> 
> // visible
> await waitFor(() => expect(el).not.toHaveStyle({ display: 'none' }))
> 
> // hide
> await waitFor(() => expect(el).toHaveStyle({ display: 'none' }))
> ```

刚查了文档，确实写复杂了...

**Comment by xiaoyao96** [comment]
感觉问题比较多，先close了...

**Pull request closed without merging**